packages/shared/src/utils/helpers/format-response.ts

```diff
@@ -93,7 +93,7 @@ export const formatIDVError = (errors: Array<TIDVErrorStatus>, status_code: stri
 
     const status: Array<TIDVErrorStatus> = [];
     errors.forEach(error => {
-        const error_key: TIDVErrorStatus = IDV_ERROR_STATUS[error].code;
+        const error_key: TIDVErrorStatus = IDV_ERROR_STATUS[error]?.code;
         if (error_key) {
             status.push(error_key);
         }
```

Comment:
shouldnt we show a default error message if error_key is not defined

Reply:
Ln: 106 handles that case. A default error code is being returned